### PR TITLE
Mark/signup form component

### PR DESCRIPTION
**Pull request by Mcorupe**

### CODE REVIEW

**Comment by signalnerve** [comment]
Make sure to request review from someone else on your team, too!

**Comment by signalnerve** [comment]
Let's hold off on review for this until #49 is in

File: frontend/src/shared-components/signup.js [hunk 1]
```diff
@@ -2,25 +2,29 @@ import React from 'react'
...
-        }
+    super(props);
+    state = {
+        username: '',
+        password: '',
+        passwordConfirmation: '',
+        email: ''
+    }
 
 this.onChange = this.onChange.bind(this);
```
Comment (shayk89) [inline frontend/src/shared-components/signup.js]: not required for arrow functions

File: frontend/src/shared-components/signup.js [hunk 2]
```diff
@@ -2,25 +2,29 @@ import React from 'react'
...
         this.setState({ [e.target.name]: e.target.value});
 }
 
 
-   render() {
+    render() {
        return (
-           <form>
+           <form onSubmit={this.onSubmit}>
                <h1> Register for PernHub! </h1>
```
Comment (shayk89) [inline frontend/src/shared-components/signup.js]: Thought this was going to be a behind the scenes joke thing....if not, I'm not explaining to any hiring managers we might demo for!

Comment (Mcorupe) [reply frontend/src/shared-components/signup.js]: I don't know i thought it was what we were going with. doenst matter to me i can swap it. maybe we bring it up during the next stand up

File: frontend/src/shared-components/login.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    if (username && password) {
+        this.props.login(username, password);
+    }
+}
+
+render() {
+    const { login } = this.props;
+    const { username, password, submitted } = this.state
+    return (
+        <form handleSubmit={this.handleSubmit}
```
Comment (signalnerve) [inline frontend/src/shared-components/login.js]: It looks like you're missing a closing `</form>` tag here

File: frontend/src/shared-components/signup.js [hunk 4]
```diff
@@ -1,42 +1,98 @@
...
+                    className="form-control"
+                />
+            </div>
+            <div className="form-group">
+                <button className="btn btn-primary btn-lg">
+                    Sign Up
+                </button>
+            </div>                    
+        </form>
+        <router>
```
Comment (signalnerve) [inline frontend/src/shared-components/signup.js]: Where is this `router` component coming from?

Comment (Mcorupe) [reply frontend/src/shared-components/signup.js]: removed. I was playing with something and overlooked it when i was editing.

**Comment by Mcorupe** [comment]
Closes #52

**Pull request merged**